models/maat/R/model2netcdf.MAAT.R

```diff
@@ -153,12 +165,27 @@ model2netcdf.MAAT <- function(rundir, outdir, sitelat = -999, sitelon = -999, st
                            newunits="Pa", missval=-999, longname="Leaf Internal CO2 Concentration", ncdims=ncdims)
       output <- var_update(sub.maat.output$cc, output, "cc", "Cc", oldunits="Pa", 
                            newunits="Pa", missval=-999, longname="Leaf Mesophyll CO2 Concentration", ncdims=ncdims)
+      
+      ## put in time_bounds before writing out new nc file
+      #length(output$var)
+      output$var[[length(output$var) + 1]] <- ncdf4::ncvar_def(name="time_bounds", units='', 
+                                                         longname = "history time interval endpoints", 
+                                                         dim=list(time_interval,time = t), 
+                                                         prec = "double")
+      output$dat[[length(output$dat) + 1]] <- c(rbind(bounds[, 1], bounds[, 2]))
       ## !!TODO: ADD MORE MAAT OUTPUTS HERE!! ##
 
     } else {
       t <- ncdf4::ncdim_def(name = "time", units = paste0("days since ", maat_run_start_date),
                             vals = 1, calendar = "standard", 
                             unlim = TRUE)  # standard calendar for leap years?  Also need to be sure we update cal depending on leap/no leap
+      bounds <- array(data=NA, dim=c(1,2))
+      bounds[,1] <- 0
+      bounds[,2] <- 1
+      time_interval <- ncdf4::ncdim_def(name = "hist_interval", 
+                                        longname="history time interval endpoint dimensions",
+                                        vals = 1:2, units="")
```

Comment:
Duplicated code. Really wish this was a small function.

Reply:
this is one is different here since it needs to a single timepoint since this is an example where we only run for one timestep.  But it could eventually be a function and provide the number or length needed